docs/.vuepress/theme/styles/index.styl

```diff
@@ -86,8 +86,16 @@ section.tabs-component-panel::before {
     margin-top: -10em;
 }
 
-.nav-links{
-    margin-right: 10%;
-	margin-left: 10%;
-	float: left;
+@media (min-width: 1060px) {
+	.nav-links{
+		margin-right: 10%;
+		margin-left: 10%;
+		float: left;
+	}
+}
+
+@media (max-width: 720px) and (min-width: 500px) {
```

Comment:
Just a nit
```suggestion
@media (min-width: 500px) and (max-width: 720px) {
```

Reply:
@DannyS03 thanks for looking at this 🥳 

Question: as my css is rusty, I'm not clear what this change fixes? Tested it locally, and it appears to be the same behavior. Or is it just to make the css look nicer (`min-width` before `max-width`)?

Reply:
Ah, right, and yes! Just to make it look nicer (i.e. best practice and being nit-picky)

Reply:
Nice makes sense